src/modal/ModalAddToFolder.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styles from './modalAddToFolder.module.css';
+import closeIcon from '../assets/_close.svg';
+
+interface ModalAddToFolderProps {
+  title: string;
+  targetName: string;
+  isModalOpen: boolean;
+  onClose: (isOpen: boolean) => void;
+}
+
+const ModalAddToFolder: React.FC<ModalAddToFolderProps> = ({
+  title,
+  targetName,
+  isModalOpen,
+  onClose,
+}) => {
+  const handleCloseModal = () => {
+    onClose(false);
+  };
+
+  return isModalOpen ? (
```

Comment:
이렇게 앞이 길고 뒤가 짧은 경우에는 그냥 짧은 부분부터 쳐내주시는 게 가독성이 좋아요!
```suggestion
  return !isModalOpen ? null : 
```